Jenkinsfile

```diff
@@ -654,6 +654,42 @@ pipeline {
                                 """
                             }
                         }
+                        stage("test-dmg") {
+                            steps {
+                                timeout(time: 5, unit: "MINUTES") {
+                                    sh '''
+                                        CHANNEL_CAPITALIZED=$(echo ${CHANNEL} | awk '{print toupper(substr($0,0,1))substr($0,2)}')
```

Comment:
for `release` this should be `""` and for `"channel"` it will be `" Channel"`, please introduce a CHANNEL_CAPITALIZED_SPACED in setEnv() instead of overwriting here